api/vector_tiles.py

```diff
@@ -58,7 +58,7 @@ def index(self, zoom, x, y):
         mapdb = cherrypy.request.app.config['DB']['map']
         d = mapdb.tables.style.data
 
-        q = sa.select([d.c.rels, d.c.allshields.label('shields'),
+        q = sa.select([d.c.rels, d.c.allrels, d.c.allshields.label('shields'),
```

Comment:
That's not the same. I want all parents for the current segment and not all children of the top relation of the current segment, for example relation [2096526](https://www.openstreetmap.org/relation/2096526):

```sql
SELECT hiking.defstyle.rels, 
    (select array(select hiking.hierarchy.parent FROM hiking.hierarchy WHERE hiking.hierarchy.child=ANY(hiking.segments.rels))) as allrels
FROM 
    hiking.defstyle, 
    hiking.segments
WHERE hiking.defstyle.id=1207
and hiking.defstyle.id=hiking.segments.id 
ORDER BY hiking.defstyle.id;
      rels       |               allrels               
-----------------+-------------------------------------
 {110075,371743} | {56703,2096526,36367,110075,371743}
```

vs.

```sql
SELECT hiking.defstyle.rels, 
    (select array(select DISTINCT hiking.hierarchy.child FROM hiking.hierarchy WHERE hiking.hierarchy.parent=ANY(hiking.defstyle.rels))) as allrels
FROM 
    hiking.defstyle
WHERE hiking.defstyle.id=1207
ORDER BY hiking.defstyle.id;
      rels       |                                          allrels                                          
-----------------+-------------------------------------------------------------------------------------------
 {110075,371743} | {36367,1158977,1158978,371743,37360,2096530,2096487,2096527,2096526,56703,110075,2096486}
```